intake/source/base.py

```diff
@@ -280,6 +280,26 @@ def hvplot(self):
         """
         return self.plot
 
+    def persist(self, name, **kwargs):
```

Comment:
Do you have a use case in mind for `name`? Is there ever a circumstance where it should not be `source.name`?